sentry-android-core/src/main/java/io/sentry/android/core/PerformanceAndroidEventProcessor.java

```diff
@@ -155,6 +156,23 @@ private void attachColdAppStartSpans(
       }
     }
 
+    // Process init
+    final long classInitUptimeMs = appStartMetrics.getClassLoadedUptimeMs();
+    final @NotNull TimeSpan appStartTimeSpan = appStartMetrics.getAppStartTimeSpan();
+    if (appStartTimeSpan.hasStarted()) {
+      final @NotNull TimeSpan processInitTimeSpan = new TimeSpan();
+      processInitTimeSpan.setStartedAt(appStartTimeSpan.getStartUptimeMs());
+      processInitTimeSpan.setStartUnixTimeMs(appStartTimeSpan.getStartTimestampMs());
+
+      processInitTimeSpan.setStoppedAt(classInitUptimeMs);
+      processInitTimeSpan.setDescription("Process Initialization");
```

Comment:
Should we add a guard in cases the app process is created a lot of time before the app is started (e.g. a notification or a service)?

Reply:
That's a good point! iOS already discards app starts longer than a certain period - let me have a look into this!

Reply:
@markushi did you have the chance to look into this?

Reply:
👍 I've added an extra check here: https://github.com/getsentry/sentry-java/pull/3159/commits/4e16c88756164c7d71fce7cb07ec514561be7ed0